<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramCfger</class>
 <widget class="QFrame" name="ProgramCfger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>260</height>
   </rect>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>600</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string>Schedule</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="friday">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Friday</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="fixedDays">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLabel" name="startTag">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="wednesday">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wednesday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="saturday">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Saturday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="tuesday">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuesday</string>
    </property>
   </widget>
   <widget class="QLabel" name="stopTag">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>110</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="stop">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>22</hour>
      <minute>22</minute>
      <second>22</second>
      <year>2000</year>
      <month>12</month>
      <day>22</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string notr="true">yyyy-M-d h:mm:ss</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="every">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Every day</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sunDay">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SunDay</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="monday">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Monday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="thursday">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Thursday</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="start">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>22</hour>
      <minute>22</minute>
      <second>22</second>
      <year>2000</year>
      <month>12</month>
      <day>22</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string notr="true">yyyy-M-d h:mm:ss</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="specified">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Specified time</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="fixed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixed duration</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sequential">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>115</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequential</string>
    </property>
   </widget>
   <widget class="QLabel" name="fixedDaysTag">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Days</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="external">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Eexternal control</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="externalDays">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLabel" name="fixedHoursTag">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Hours</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="fixedHours">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>23</number>
    </property>
   </widget>
   <widget class="QLabel" name="fixedMinutesTag">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Minutes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="fixedMinutes">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
   </widget>
   <widget class="QLabel" name="fixedSecondsTag">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Seconds</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="fixedSeconds">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
   </widget>
   <widget class="QLabel" name="externalDaysTag">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Days</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="externalHours">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>23</number>
    </property>
   </widget>
   <widget class="QLabel" name="externalHoursTag">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Hours</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="externalMinutes">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
   </widget>
   <widget class="QLabel" name="externalMinutesTag">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Minutes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="externalSeconds">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
   </widget>
   <widget class="QLabel" name="externalSecondsTag">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Seconds</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
